dapp/src/components/buySell/SwapCurrencyPill.js

```diff
@@ -282,7 +282,7 @@ const SwapCurrencyPill = ({
   const roundTo2to6Decimals = (value) => {
     return formatCurrencyMinMaxDecimals(value, {
       minDecimals: 2,
-      maxDecimals: 6,
+      maxDecimals: 20,
```

Comment:
Function name is wrong if this change is required. Also why is this change required?

Reply:
![image](https://user-images.githubusercontent.com/18202625/136130948-a5067d85-eb2f-451c-8986-535e598b0ec6.png)

Reply:
It is for "Max Including Dust #652" requirements...